css/Styles.css

```diff
@@ -1,28 +1,485 @@
+    width: 80px;
+    transition: all 0.5s;
+    cursor: pointer;
+}
+
+::placeholder {
+    color: rgb(182, 180, 180);
+    font-family:"Work Sans", sans-serif;
+    font-size: 14px;   
+}
+
+nav input[type=text] {
+    padding: 7px;
+    background-color: #656b72;
+    border-radius: 7px;  
+}
+.border {
+    border-radius: 90%;
+}
+
+.caret {
+    line-height: 42px;
+    float: right;
+}
 
-.five.columns {
-    background: green;
+.parent:hover .caret {
+    transform: rotate(90deg);
+    transition: 500ms ease;
 }
 
-.two.columns {
-    background: red;
+h1 {
+    color: white;
+    font-size: 60px;
 }
 
-.four.columns {
-    background: yellow;
+.grey-text {
+    color: #8b929c;
+    font-size: 21px;
 }
 
-.six.columns {
-    background: blue;
+.text-center {
+    text-align: center;
 }
 
-.one.column {
-    background: orangered;
+.underline {
+        color: rgb(199, 198, 198);
 }
 
-.ten.columns {
-    background:pink;
+.underline:hover {
+        color: #fff;
 }
 
-.eight.columns {
-    background: purple;
-}
\ No newline at end of file
+.form-div {
+    background-color: white;
+    border: #fff;
+    border-radius: 7px;
+    height: 350px;
+    width: 350px;
+    padding: 50px 20px;
+    position: absolute;
+    right: 20%;
+    bottom: 4%;
+}
+
+.text-div-one {
+    position: absolute;
+    height: 100px;
+    width: 40%;
+    right: 52%;
+    bottom: 55%;
+}
+
+.green-button {
+    height: 60px;
+    width: 400px;
+    border-radius: 8px;
+    background-color: rgb(27, 192, 27);
+    color: white;
+    font-size: 16px;
+    border-style: none;
+}
+
+.green-button:hover {
+    background-color: rgb(10, 134, 10);
+    transition: all 250ms ease-in-out;
+}
+
+.form {
+    width: 370px;
+    height: 25px;
+    margin-top: 5px;
+    margin-bottom: 15px;
+}
+
+.form-label {
+    padding: 5px;
+    padding-bottom: 5px;
+}
+
+form input[type=text] {
+    padding: 7px;
+    background-color: #ffffff;
+    border-radius: 7px; 
+    border-width: 2px; 
+}
+
+form input[type=text]:focus {
+    padding: 7px;
+    background-color: #ffffff;
+    border-radius: 7px;
+    border-color: rgb(0, 183, 255); 
+    outline: none;
+    box-shadow: rgb(0, 110, 255) 0 0 10px;
+}
+
+.section-two {
+    position: relative;
+    width: 100%;
+    background-color: white;
+    border-style: solid 1px black;
+    margin: auto;
+    margin: 150px;
+}
+
+.div-box {
+    position: relative;
+    height: 99%;
+    width: 100%;
+    box-shadow: rgba(70, 69, 69, 0.404) 0 0 15px;
+    border-radius: 10px;
+}
+
+.text-div-two {
+    color: black
+    width: 300px;
+    font-size: 18px;
+    position: absolute;
+    bottom: -5%;
+}
+
+.text-div-three {
+    color: black;
+    width: 350px;
+    text-align: center;
+    line-height: 13px;
+    display: block;
+    margin: auto;
+
+}
+
+.blue-button {
+    color: white;
+    background-color: rgb(0, 162, 255);;
+    height: 50px;
+    width: 150px;
+    border-radius: 7px;
+    border-style: none;
+    font-size: medium;
```

Comment:
Medium is a little bit generic and could be interpreted differently by different browsers, can we be a bit more specific on the size?